docs/housing-counselor-data.md

```diff
@@ -0,0 +1,163 @@
+# Housing Counselor Data Processing
+
+The [Find a Housing Counselor tool](https://www.consumerfinance.gov/find-a-housing-counselor/) gets its data from the US Department of Housing and Urban Development. HUD provides an API to their list of approved counseling agencies.
+
+We generate the JSON data for the Find a Housing Counselor tool using the `hud_generate_json` management command. This page describes how that command works to transform HUD data to the JSON file format we use for the FAHC tool, including the following steps:
```

Comment:
How would you feel about adding links throughout this document to the Python source files being referenced, e.g. [`hud_generate_json`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/housing_counselor/management/commands/hud_generate_json.py)? This would make it easier for developers to follow along in the code when reading this document.